e2e/scripts/docker/definition.test

```diff
@@ -0,0 +1,30 @@
+#!/singularity/test
```

Comment:
This test is a good example of where a loop / table-driven test would be great to have. Having to copy and paste all the test code just for different examples seems a little problematic.